File: JavaScript/abc027A.js
  function Main(input) {
      const a = input[0].split(' ');
      if(a[0] == a[1]){
        console.log(a[2]);
      }else if(a[0] == a[2]){
        console.log(a[1]);
      }else{
        console.log(a[0]);
      }
  }
  
  Main(require("fs").readFileSync("/dev/stdin", "utf8"));
  

File: C#/abc027A.cs
using System;

class main{
  public static int Main(){
    string[] a = Console.ReadLine().Split(' ');
    if(a[0] == a[1]){
        Console.WriteLine(a[2]);
    }else if(a[1] == a[2]){
        Console.WriteLine(a[0]);
    }else if(a[0] == a[2]){
        Console.WriteLine(a[1]);
    }
    return 0;
  }
}
